packages/studio/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "baseUrl": "./",
     "rootDir": "api",
-    "tsBuildInfoFile": "dist/api/tsconfig.tsbuildinfo",
+    "tsBuildInfoFile": "tsconfig.tsbuildinfo",
```

Comment:
Is this one specified just because studio is structured a bit differently?

Reply:
Yeah that's the reason. I think if I totally removed the option it was in somewhere slightly different to the place we now put it in other packages. Hopefully this package can be ripped out in to it's own repo not too far in the future.